README.md

```diff
@@ -70,10 +72,13 @@ key = <YOUR LOGDNA INGESTION KEY>
 ```
 
 #### Configuration File Options
-* `logdir`: sets the paths that the agent will monitor for new files. Multiple paths can be specified, separated by `,`. Supports glob patterns + specific files. By default this option is set to monitor `.log` and extensionless files under `/var/log/`.
-* `exclude`: sets files to exclude that would otherwise match what's set in `logdir`. Multiple paths can be specified, separated by `,`. Supports glob patterns + specific files
+* `logdir`: sets the paths that the agent will monitor for new files. Multiple paths can be specified, separated by `,`. Supports glob patterns + specific
```

Comment:
is there an order we have for these options- or an order that would make most sense?

Reply:
I usually order them in the order that they are in the code, but there's really no rule.  Alphabetizing is another option.  We will be re-writing all of these docs in another PR.